newrelic/provider.go

```diff
@@ -184,3 +184,19 @@ func providerConfigure(data *schema.ResourceData, terraformVersion string) (inte
 
 	return &providerConfig, nil
 }
+
+func getInfraAPIURL(data *schema.ResourceData) string {
+	old, oldOk := data.GetOk("infra_api_url")
+	new, newOk := data.GetOk("infrastructure_api_url")
```

Comment:
I think `new` is a keyword in Go. At a minimum it's a function: `new()`

Might want to use `newURL` maybe?